apps/website/src/pages/contact-us.tsx

```diff
@@ -20,43 +19,87 @@ const ContactUsPage: NextPage = () => {
 
       {/* Grow the last section to cover the page */}
       <Section
-        className="flex min-h-[85vh] flex-grow items-center"
+        className="flex min-h-[85vh] flex-grow"
         containerClassName="flex flex-col items-center text-center text-alveus-green"
       >
-        <Heading className="text-6xl">Contact Us</Heading>
+        <div className="my-auto flex flex-col items-center">
+          <Heading className="text-6xl">Contact Us</Heading>
+          <p className="my-3 text-xl font-semibold">
+            We are not openly hiring.
+          </p>
 
-        <p className="my-3 text-xl font-semibold">
-          <Link href="mailto:maya@alveussanctuary.org">
-            maya@alveussanctuary.org
-          </Link>
-        </p>
+          <p className="my-3 text-xl font-semibold">
+            For business inquiries: <br />
+            <Link href="mailto:TeamAlveus@unitedtalent.com">
+              TeamAlveus@unitedtalent.com
+            </Link>
+          </p>
 
-        <p>You can send in a resume to maya@alveussanctuary.org.</p>
-        <p>You will be put on a list for future reference.</p>
+          <div className="flex flex-wrap gap-x-6">
+            <p className="my-3 text-xl font-semibold">
+              For merch inquiries: <br />
+              <Link href="https://shopalveus.org/pages/contact" external>
+                shopalveus.org/pages/contact
+              </Link>
+            </p>
 
-        <p className="my-3 text-xl font-semibold">We are not openly hiring.</p>
+            <p className="my-3 text-xl font-semibold">
+              For plushie inquiries: <br />
+              <Link href="https://youtooz.com/pages/contact-us" external>
+                youtooz.com/pages/contact-us
+              </Link>
+            </p>
+          </div>
```

Comment:
this aligns weird on mobile, probably needs some sort of center aligning or flex-grow or smth. i'm too lazy to open the code though Smile